### DISCUSSION TITLE
Website Sign up Issue

### DISCUSSION THREAD

**Discussion by jeff-hykin** [npm/feedback#617] [Website] [answered]
Every time I try to setup an account (even after different usernames, different emails, different passwords, different browsers, different devices, and different ip addresses) I'm continually getting this error.
<img width="770" alt="Screen Shot 2021-12-05 at 7 17 51 AM" src="https://user-images.githubusercontent.com/17692058/144748165-f0e4adbf-4f16-4009-aa29-b68701de1df3.png">

The diagnostic number changes, for example here's another one: `diagnostics id: 6b8de5814c396776~86a9`

I don't see anything similar in the discussions, or generally online, and the dev tools don't add much information.

The emails are gmail accounts, the passwords and usernames follow the character requirements.

<img width="510" alt="Screen Shot 2021-12-05 at 7 20 02 AM" src="https://user-images.githubusercontent.com/17692058/144750365-aded8515-2e63-4e3d-9d56-10188a6ee3c8.png">

**Comment by MylesBorins** [comment, marked as answer]
@jeff-hykin if you are still running into this issue please open a ticket with [npm support](https://npmjs.com/support)
